cafebabel/core/helpers.py

```diff
@@ -73,3 +73,9 @@ def absolute(url):
     if url.startswith('/'):
         url = url[1:]
     return f'{request.url_root}{url}'
+
+
+def allowed_file(filename):
```

Comment:
This makes me wonder about the dangers of uploading the correct file extension but faked with a different mimetype.

Reply:
It should be handled at the Apache level + it should not be executed by the browser anyway.

Reply:
I'm aware the browser would prevent from user failure. We shall check for security improvement in our forms pretty much everywhere indeed.